frontend/components/droplets/lessons/lesson-renderer.tsx

```
@@ -58,6 +58,133 @@ interface HighlightResponseItem {
+                if (item.type === "text") {
+                  let text = item.text || "";
+                  // Apply styles
+                  if (item.styles?.bold) text = `<strong>${text}</strong>`;
+                  if (item.styles?.italic) text = `<em>${text}</em>`;
+                  if (item.styles?.underline) text = `<u>${text}</u>`;
+                  if (item.styles?.code) text = `<code>${text}</code>`;
+                  return text;
+                }
+                return "";
+              })
+              .join("") || "";
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)